orc8r/cloud/go/services/accessd/storage/storage_blobstore_test.go

```diff
@@ -85,10 +87,13 @@ func TestAccessdBlobstore_ListAllIdentity(t *testing.T) {
 	// store.ListKeys fails
 	blobFactMock = &mocks.BlobStorageFactory{}
 	blobStoreMock = &mocks.TransactionalBlobStorage{}
+
 	blobFactMock.On("StartTransaction", mock.Anything).Return(blobStoreMock, nil).Once()
 	blobStoreMock.On("Rollback").Return(nil).Once()
-	blobStoreMock.On("ListKeys", mock.Anything, astorage.AccessdDefaultType).
-		Return([]string{}, someErr).Once()
+	blobStoreMock.On("Search",
```

Comment:
When you do multi-line function args, the open and close parens should be on their own lines. So e.g.

```
blobStoreMock.On(
"Search",
filter,
loadcriteria,
)
```